integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```
@@ -68,7 +68,8 @@ expected_metrics:
       node_name: .*
       namespace_name: .*
     monitored_resource: gce_instance
-    type: workload.googleapis.com/aerospike.namespace.scan.count
+    type: workload.googleapis.com/aerospike.namespace.scan.count # This aerospike metric is broken for aerospike 6.X - see issue raised https://github.com/open-telemetry/opentelemetry-collector-contrib#21834
```

The comment really pertains to the next line (`optional: true`). So I'd put that comment right before that line. You could also put it on the line instead…

Also note that the link is broken. Fixed in the suggestion below…
```suggestion
    type: workload.googleapis.com/aerospike.namespace.scan.count
    # This aerospike metric is broken for aerospike 6.X - see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21834
```